toolset/benchmark/test_types/framework_test_type.py

```diff
@@ -146,7 +151,8 @@ def get_current_world_table(self):
 
         if database_name == "mysql":
             try:
-                db = MySQLdb.connect("TFB-database", "benchmarkdbuser", "benchmarkdbpass", "hello_world")
+                db = MySQLdb.connect("TFB-database", "benchmarkdbuser",
+                                     "benchmarkdbpass", "hello_world")
```

Comment:
Now might be a good time for a constants.py

Reply:
I tried this out and disliked it here. These values will never change and you'll never get a meaningful constant that is as short as these strings, magic though they may be.